ci-operator/step-registry/aks/provision/aks-provision-commands.sh

```diff
@@ -21,6 +21,7 @@ AKE_CREATE_COMMAND=(
     --node-count "$AKS_NODE_COUNT"
     --load-balancer-sku "$AKS_LB_SKU"
     --os-sku "$AKS_OS_SKU"
+    --location "$LEASED_RESOURCE"
```

Comment:
Accordingly shall we update aks-deprovision step too?

Reply:
The `az aks delete` command does not have a flag to specify the location. It seems Azure automatically detects the location of the existing cluster for us.

Reply:
got it.